pkg/asset/agent/installconfig.go

```diff
@@ -114,6 +120,36 @@ func (a *OptionalInstallConfig) validateSupportedPlatforms(installConfig *types.
 	return allErrs
 }
 
+func (a *OptionalInstallConfig) validateReleaseArch(installConfig *types.InstallConfig) field.ErrorList {
```

Comment:
@andfasano I'm not sure if this is exactly what you meant by this [comment](https://github.com/openshift/installer/pull/7349#discussion_r1344319648)? Once I took a 2nd look, it made sense to move all of the releaseimage arch logic to its own validation.